source/process.adoc

```diff
@@ -184,7 +184,7 @@ The Specification Document and related technical artifacts must be developed by
 [[efsp-tck]]
 ==== Technology Compatibility Kits
 
-There is exactly one TCK project under an Open Source License for each Specification Version.
+There is exactly one TCK under an Open Source License for each Specification Version.
```

Comment:
Should we qualify this with "under an approved Open Source License"?  Or, since it's capitalized, is it assumed that the definitions in the EFSP take precedence?

Reply:
The capitalization indicates that the defined term is intended.